TwitchLib.Client.Models/Builders/GiftedSubscriptionBuilder.cs

```diff
@@ -8,7 +7,7 @@ public sealed class GiftedSubscriptionBuilder : IBuilder<GiftedSubscription>, IF
     {
         private readonly List<KeyValuePair<string, string>> _badges = new List<KeyValuePair<string, string>>();
         private readonly List<KeyValuePair<string, string>> _badgeInfo = new List<KeyValuePair<string, string>>();
-        private string _color;
+        private Color _color;
```

Comment:
so we actually had this originally at some point but it required declaring a dependency on System.Drawing, which iirc was breaking things in unity. Not sure if @Syzuna has additional context on this. If y'all think we're good here on this, its cool with me

Reply:
IIRC Unity isn't the only thing that has issues with System.Drawing - certain other targets (and Native AOT) had certain cases where System.Drawing required replacing it with SkiaSharp.

Seconding the suggestion that it's best to try to avoid referencing it.

Reply:
if the dependency on System.Drawing is such a problem we should solve it somehow
maybe we could address this in the new Issue instead of the PR


There may be solutions:

- Ignore it and continue to use `Color` from System.Drawing
- Replace everything with the string HeColor`
- Replace everything with a custom `struct Color` string
- Replace everything with `Color` from SkiaSharp (or other alternative)

Reply:
Imo I think the best route is probably just a string for now. Let the implementations using the library determine how to handle the color (presumably most will convert to System.Drawing.Color).

Reply:
what is your opinion on it @Syzuna?

Reply:
if we use a string, we could create an interface
```en
interface IHexColorProperty // this needs a better name
{
     string HexColor { get; }
}
```
so people could then easily create an extension method for parsing color on "models"

Reply:
Ya, that seems reasonable I think. I'll let you decide the correct implementation, I just wanted to steer clear of System.Drawing.